examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,20 +3,21 @@
  * Licensed under the MIT License.
  */
 
-import { SchemaBuilder, Typed, leaf } from "@fluid-experimental/tree2";
+import { ProxyNode, SchemaBuilder } from "@fluid-experimental/tree2";
 
-const builder = new SchemaBuilder({ scope: "inventory app" });
+const builder = new SchemaBuilder({
+	scope: "com.contoso.app.inventory",
+});
 
-export const part = builder.struct("Contoso:Part-1.0.0", {
-	name: leaf.string,
-	quantity: leaf.number,
+export const part = builder.struct("Part", {
+	name: builder.string,
+	quantity: builder.number,
 });
 
-export const inventory = builder.struct("Contoso:Inventory-1.0.0", {
-	parts: builder.sequence(part),
+export const inventory = builder.struct("Inventory", {
```

Comment:
why not use SchemaBuilder.list here?

Reply:
I didn't notice '.list()' was in!  This makes my day!